src/utils/options.ts

```diff
@@ -209,6 +214,7 @@ export {
   agentPort,
   connConnectTime,
   recoveryCodeFile,
+  recoveryCodeOutFile as recoveryCodeOutFile,
   passwordFile,
```

Comment:
What's with the `as` assignment here?

Reply:
That's strange, I'm not sure why that's there. I'll look into it.

Reply:
It seems it might've been my LSP decided to rename the variable to itself, that's very odd. Fixed in the next commit I'll be pushing up soon.